modular-docs-manual/content/topics/module_mod-docs-concept-examples.adoc

```diff
@@ -1,8 +1,8 @@
 [id="modular-docs-concept-examples"]
 = Concept module examples
 
-link:https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html-single/creating_red_hat_process_automation_manager_business_applications_with_spring_boot/index#bus_app_business-applications[Red Hat Process Automation Manager business applications]
+link:https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.13/html/creating_red_hat_process_automation_manager_business_applications_with_spring_boot/bus_app_business-applications[Red Hat Process Automation Manager business applications]
```

Comment:
I get a 404 at this link. 
I think it moved here. https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.13/html/integrating_red_hat_process_automation_manager_with_other_products_and_components/assembly-springboot-business-apps

Reply:
+1

Reply:
Fixed. Thank you @neal-timpe for spotting this.